pkg/k8s/k8s.go

```diff
@@ -22,6 +23,9 @@ const (
 
 	// certPath path in the filesystem to the ca.crt
 	certPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+	// waitRetry is the time to wait between retries for a readiness checking
+	waitRetry = 2 * time.Second
```

Comment:
If there is a reason 2 seconds was picked we should add it to this comment. This is helpful for debugging in the future if 2 seconds causes issues, we have some context to go off of in terms of determining a better value.

Reply:
2 here is just an arbitrary number to wait between the checks. @jrmanes @smuu  what number do you think is the best to be used here?

Reply:
Thanks Jose